packages/block-editor/src/components/block-list/use-in-between-inserter.js

```diff
@@ -32,10 +33,6 @@ export function useInBetweenInserter() {
 	const { showInsertionPoint, hideInsertionPoint } =
 		useDispatch( blockEditorStore );
 
-	const delayedShowInsertionPoint = useDebounce( showInsertionPoint, 500, {
-		trailing: true,
-	} );
```

Comment:
I reverted all of #44325, but we can probably keep this bit. Only the `IntersectionObserver` debouncing needs to be reverted I think.